lib/tasks/export.rake

```diff
@@ -0,0 +1,31 @@
+namespace :export do
+  desc "Export Alumni"
+  # usage:  rake export:alumni
+  task alumni: :environment do
+    require 'csv'
+
+      a = Alumn.all
+
+      CSV.open("/tmp/alumni.csv", "a+") do |csv|
+        csv << [ "id","box","lname","fname","year","graduate","photos","oversize" ]
+        a.each { |row| csv << [ row.id, row.lname, row.fname, row.box, row.year, row.graduate, row.photos, row.oversize ] }
```

Comment:
Tiny thing, but I think `Alum.find_each` (See [#find_each](http://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each)) will be more efficient than `a = Alum.all` (make a bunch of `Alum` objects in memory, stored in `a`) followed by `a.each` to loop over them.